staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -379,9 +379,17 @@ func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	localError := make(chan struct{})
 	remoteDone := make(chan struct{})
 
+	drainRemoteStream := func() {
+		_, err := io.ReadAll(dataStream)
```

Comment:
avoid ReadAll just to discard the results

prefer something like `io.Copy(ioutil.Discard, dataStream)`

Reply:
fixed.